_episodes/07-branches.md

```diff
@@ -80,13 +101,13 @@ $ git branch
 ~~~
 {: .output}
 
+Before we used the `checkout` command to checkout a commit using
+commit hashs or `HEAD`. This is similar but now we have given our most
```

Comment:
'This is similar but now we have given our most recent commit in the branch a name, the branch name.' - I find something about this statement a bit confusing

Maybe something like:
This is similar but now we give `checkout` the branch name, which is a nickname for the most recent commit.
Not sure that is any good either.  I've never thought of a branch as a name for the most recent commit before so I might be way off base here